Control cross tenant replication (#16351)
hashicorp/terraform-provider-azurerm: website/docs/r/storage_account.html.markdown

* `account_replication_type` - (Required) Defines the type of replication to use for this storage account. Valid options are `LRS`, `GRS`, `RAGRS`, `ZRS`, `GZRS` and `RAGZRS`. Changing this forces a new resource to be created when types `LRS`, `GRS` and `RAGRS` are changed to `ZRS`, `GZRS` or `RAGZRS` and vice versa.

> * `cross_tenant_replication_enabled` - (Optional) Allow or disallow cross tenant replication. Defaults to `true`.

Reviewer: fwiw we tend to phrase these as "Should {thing}?"

```suggestion
* `cross_tenant_replication_enabled` - (Optional) Should cross Tenant replication be enabled? Defaults to `true`.
```
